### PR TITLE
Add info about supported Plone versions to README

### PR DESCRIPTION
**Pull request by svx**
Fixes: #

Improves: Our README template, by adding information about supported Plone versions.

Changes proposed in this pull request:

- Update README template with info about supported Plone versions

-

-

### CODE REVIEW

**Comment by gforcada** [comment]
@svx I would rebase and squash all commits into a single one (I think github UI allows you to do that(?)) so back/future-porting will be easier :-)

**Comment by svx** [comment]
@gforcada yes, but if I do that, I will merge, so you have to do it, just click on the arrow next to 'merge pull request' and choose one :)

**Comment by svx** [comment]
BTW, if that PR gets merged we need up update our bob.templates

**Comment by gforcada** [comment]
@svx so far I didn't see anyone opposing the decision, so let's merge it and create a ticket on bobtemplate.plone :-)

File: about/documentation_styleguide.rst [hunk 1]
```diff
@@ -112,6 +112,12 @@ This is an example of how a README.rst should look like:
     - Works out of the box, but can also be customized.
       After installation, you will find a new item in your site control panel where to set various options.
 
+    Supported Plone Versions
+    ------------------------
+
+    - Plone 5.0
+    - Plone 4.4
```
Comment (jensens) [inline about/documentation_styleguide.rst]: https://plone.org/download/ says 4.3

Comment (svx) [reply about/documentation_styleguide.rst]: Yes, evil typo, fixed ! Thanks !

**Pull request merged**